threads/views.py

```
@@ -173,4 +174,4 @@ def get_reactions_count(self, request, *args, **kwargs):
             .values('reaction')
             .annotate(count=Count('id'))
         )
-       return Response(reaction_counts)
+       return Response(reaction_counts)
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)